contracts/modules/credit/SpigotedLine.sol

```diff
@@ -68,12 +69,11 @@ contract SpigotedLine is ISpigotedLine, LineOfCredit, ReentrancyGuard {
 
     /**
 
-   * @notice - Claims revenue tokens from Spigot attached to borrowers revenue generating tokens
-               and sells them via 0x protocol to repay credits
+   * @notice - Claims revenue tokens from the Spigot, trades them for credit tokens via a Dex (Ox) and sends the credit tokens to a Lender to repay debt.
    * @dev    - callable `borrower` + `lender`
    * @notice see _repay() for more details
-   * @param claimToken - The revenue token escrowed by Spigot to claim and use to repay credit
-   * @param zeroExTradeData - data generated by 0x API to trade `claimToken` against their exchange contract
+   * @param claimToken - The revenue token escrowed by a Spigot to claim and then trade via a Dex for a credit token to use to repay credit
```

Comment:
can we change end to "repay debt" i think thats more accurate

Reply:
also can you make it "DEX" or "dex" but def not "Dex". Same on line below

Reply:
repay debt --> repay credit done and I'll watch out for this elsewhere
I didn't see Dex anywhere here but I added 'dex' before "Ox API"